airbyte-integrations/connectors/source-mongodb-v2/src/main/java/io/airbyte/integrations/source/mongodb/cdc/MongoDbDebeziumPropertiesManager.java

```diff
@@ -106,21 +106,16 @@ public static String normalizeName(final String name) {
    * removing any values accidentally copied and pasted from the MongoDB Atlas UI.
    *
    * @param config The connector configuration.
-   * @param useSecondary Whether to use the secondary for reads.
    * @return The connection string.
    */
-  public static String buildConnectionString(final JsonNode config, final boolean useSecondary) {
+  public static String buildConnectionString(final JsonNode config) {
     final String connectionString = config.get(CONNECTION_STRING_CONFIGURATION_KEY)
         .asText()
         .trim()
         .replaceAll(DOUBLE_QUOTES_PATTERN, "")
         .replaceAll(CREDENTIALS_PLACEHOLDER, "");
     final StringBuilder builder = new StringBuilder();
     builder.append(connectionString);
-    builder.append("?retryWrites=false&provider=airbyte&tls=true");
```

Comment:
I wonder if provider is something we want to leave.
I'm leaning No as this isn't a valid param

Reply:
@rodireich yeah, I'd also lean "no" since we know it isn't supported in Atlas at least.